app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    file_to_copy_name = command[1]
+    new_file_name = command[2]
```

Comment:
Make sure you check that the cp input command is correct.

Check the length of the splitted command to make sure it consists of three parts. Better use unpacking instead of indexing